www/main/common/sttm-ui/custom-tile/CustomBgTile.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CustomBgTile = ({ customBg, onApply, onRemove }) => {
```

Comment:
![image](https://user-images.githubusercontent.com/2982784/113954789-61d05600-97cf-11eb-9f31-084993fc51a8.png)
- In production we show the thumbnail of the image but this change shows blank image. 
- Also in prod we can't delete the image unless we're on a different theme. We can delete the custom theme anytime here. 
- The custom theme isn't showing up in the ceremonies selection for current theme.